src/openMVG/calibration/bestImages.hpp

```diff
@@ -10,62 +10,64 @@ namespace calibration{
 /**
  * @brief This function computes cell indexes per image.
  *
- * @param[in] imagePoints
- * @param[out] cellIndexesPerImage
- * @param[in] imageSize
- * @param[in] calibGridSize
+ * @param[in] imagePoints Coordinates of the 2D points in each image of the sequence.
+ * @param[in] imageSize The size of the image.
+ * @param[in] calibGridSize The number of cells per edge.
+ * @param[out] cellIndexesPerImage The id of the cell for each point of the image sequence.
  */
 void precomputeCellIndexes(const std::vector<std::vector<cv::Point2f> >& imagePoints,
-                           std::vector<std::vector<std::size_t> >& cellIndexesPerImage,
-                           const cv::Size& imageSize, const std::size_t calibGridSize);
+                           const cv::Size& imageSize,
+                           const std::size_t calibGridSize,
+                           std::vector<std::vector<std::size_t> >& cellIndexesPerImage);
 
 /**
  * @brief This function counts the number of points in each cell of the grid.
  *
- * @param[in] imagesIndexes
- * @param[in] cellIndexesPerImage
- * @param[out] cellsWeight
- * @param[in] calibGridSize
+ * @param[in] imagesIndexes The images id of the sequence.
+ * @param[in] cellIndexesPerImage The id of the cell for each point of the image sequence.
+ * @param[in] calibGridSize The number of cells per edge.
+ * @param[out] cellsWeight The number of points for each cell id.
  */
 void computeCellsWeight(const std::vector<std::size_t>& imagesIndexes,
                         const std::vector<std::vector<std::size_t> >& cellIndexesPerImage,
-                        std::map<std::size_t, std::size_t>& cellsWeight, const std::size_t calibGridSize);
+                        const std::size_t calibGridSize,
+                        std::map<std::size_t, std::size_t>& cellsWeight);
 
 /**
  * @brief This function computes the score of each image.
  *
- * @param[out] imageScores
- * @param[in] remainingImagesIndexes
- * @param[in] cellIndexesPerImage
- * @param[in] cellsWeight
+ * @param[in] remainingImagesIndexes The ids of the images in which we pick up the maxCalibFrames with the best score.
```

Comment:
not really clear to me. From the name it seems like it contains the indices that have not been selected. From the explanation it seems the contrary. The same phrase appears 3 times, so if we change we need to change everywhere.